.ci/releaseBuild.yml

```
@@ -98,6 +98,13 @@ stages:
           **\*.psm1
         useMinimatch: true
 
+    - template: Sbom.yml@ComplianceRepo
+      parameters:
+        BuildDropPath: $(signOutPath)
+        Build_Repository_Uri: 'https://github.com/powershell/powershellget'
+        PackageName: 'PowerShellGet'
+        PackageVersion: $moduleVersion

```

```suggestion
        PackageVersion: '$(moduleVersion)'

```

This is YAML variable which is setup in line 75.